src/super_gradients/training/utils/sg_trainer_utils.py

```diff
@@ -329,3 +333,18 @@ def handle_exception(exc_type, exc_value, exc_traceback):
         logger.error("Uncaught exception", exc_info=(exc_type, exc_value, exc_traceback))
 
     sys.excepthook = handle_exception
+
+
+def parse_args(cfg, arg_names: Union[List[str], callable]) -> dict:
```

Comment:
I don't think this is the desired behaviour....
We shouldn't override with Nones

Reply:
fixed